static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -27,13 +32,27 @@ define([
                 var questionText = question.content;
 
                 if (questionText && atomId) {
+                    var questionElementClicked = document.getElementById(questionId);
```

Comment:
Is this the [button](https://github.com/guardian/frontend/pull/16976/files#diff-d972eeefc72609927a5346d4b85a106bR24)?